### DISCUSSION TITLE
Language choice must be in the very first page of the first-run setup

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#19071] [Ideas] [answered]
As simple as it is.
Right here, I guess
![image](https://user-images.githubusercontent.com/86934170/178224264-08e96fb0-83c3-4677-b164-b5f99150acb7.png)

**Comment by frenzibyte** [comment]
Will mention that osu!(lazer) will eventually detect the user's system language and use it as its default. See https://github.com/ppy/osu-framework/issues/5113.

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/19107
